samples/tfm/tfm_hello_world/README.rst

```diff
@@ -12,7 +12,7 @@ A simple sample based on Hello World that demonstrates adding Trusted Firmware-M
 Overview
 ********
 
-This sample uses the Platform Security Architecture (PSA) API to calculate a SHA-256 digest.
+This sample uses the Platform Security Architecture (PSA) API to calculate a SHA-256 digest and read two FICR registers.
```

Comment:
The read function is not strictly PSA. This text would actually be more helpful if it said something like

> This sample calls secure services in TFM image to calculate ...

Reply:
Changed to `This sample uses the Platform Security Architecture (PSA) API to calculate a SHA-256 digest and the TF-M platform read service to read two FICR registers.`